src/rashdf/plan.py

```diff
@@ -1,9 +1,79 @@
 from .geom import RasGeomHdf
-
+from .base import RasHdf
+from typing import Dict
 from geopandas import GeoDataFrame
 
 
-class RasPlanHdf(RasGeomHdf):
+class RasPlanHdf(RasHdf):
+    PLAN_INFO_PATH = "Plan Data/Plan Information"
+    PLAN_PARAMS_PATH = "Plan Data/Plan Parameters"
+    PRECIP_PATH = "Event Conditions/Meteorology/Precipitation"
+    RESULTS_UNSTEADY_PATH = "Results/Unsteady"
+    RESULTS_SUMMARY_PATH = f"{RESULTS_UNSTEADY_PATH}/Summary"
```

Comment:
Maybe we should name this `RESULTS_UNSTEADY_SUMMARY_PATH` since there could be Steady Summary attributes as well?